source/agora/common/Amount.d

```diff
@@ -260,11 +260,27 @@ public struct Amount
     ***************************************************************************/
 
     pragma(inline, true)
-    public void percentage (ubyte percentage)
+    public bool percentage (ushort pct)
```

Comment:
because now we have a method called 
```D
    pragma(inline, true)
    public Amount mustPercentage (ushort pct)
    {
        percentage(pct) || assert(0);
        return this;
    }
```

and this method calls the percentage method, so I had to change the percenatege parameter to pct to be consistent

Reply:
Well `this.percentage(percentage)` would have worked but I see your point now.